site/en/docs/handbook/extensions/index.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Extensions docs
+layout: 'layouts/doc-post.njk'
+date: 2021-02-02
+updated: 2021-02-02
+---
+
+## Line wrapping {: #line-wrapping }
+
+Extensions docs are wrapped at 100 characters. New content and updates to existing content must
+follow this convention save for the following exceptions:
+
+* The existing page is already wrapped at another offset.
+    * Continue wrapping the document at the current offset.
+* Rewrapping would introduce significant noise to a code review.
+    * Try to limit line wrapping normalization to the text blocks your touch in a PR.
+
+## Header IDs {: #header-ids }
+
+Header IDs are automatically generated based on the content of the header. This means that renaming
+a header will also change the header's ID and break existing links to that header.
+
+In order to make our documentation more resilient to change, authors should manually headers IDs.
+
+```md
+// Do
+### Common use cases {: #use-cases }
+
+// Don't do
+### Common use cases
+```
+
+Header IDs should be short and meaningful. Try to use descriptive names that unambiguously identify
+the content of the section rather than a [kebab cased][kebab-case] version of the header's text
+value.
+
+## Link conventions {: #links }
+
+### Footer links
+
+By convention, extensions docs strongly prefer named footer links over inline links. Named links
+have a couple of advantages over inline links.
+
+- **Named links are typically shorter.** This makes it much easier to read and
+work with source Markdown file. 
+- **Named links are easier to maintain.** Authors can edit one link definition and update all uses
+of that link links across a given document. 
+
+```md
+// Do
+Lorem ipsum dolor [sit amet][mdn-global], consectetur adipiscing elit.
+Quisque aliquam rutrum pellentesque. Ut tincidunt purus ex, eget 
+congue lacus aliquet quis.
+
+[mdn-global]: https://developer.mozilla.org/en-US/docs/Glossary/Global_object
+
+// Don't do
+Lorem ipsum dolor [sit
+amet](https://developer.mozilla.org/en-US/docs/Glossary/Global_object),
+consectetur adipiscing elit. Quisque aliquam rutrum pellentesque. Ut
+tincidunt purus ex, eget congue lacus aliquet quis.
+```
+
+### Internal links {: #internal-links }
+
+When linking to other resources on `developer.chrome.com`, use the absolute path of the page rather
```

Comment:
Is this a handbook for our writers or external extension writers? This is not only contrary to our handbook, we also have a ci test that flags it in PRs.

Reply:
Commit 39cc94f clarifies the pathing guidance. I think we can resolve this conversation.